packages/app-project/pages/projects/[owner]/[project]/classify/workflow/[workflowID]/index.js

```diff
@@ -0,0 +1,7 @@
+export { default } from '@screens/ClassifyPage'
+
+export function getServerSideProps({ params, req, resp }) {
```

Comment:
```suggestion
export function getServerSideProps({ params, req, res }) {
```
See https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering